weatherApp/Features/SearchLocation/Data/CoreDataModels/CityCoreData+CoreDataClass.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  CityCoreData+CoreDataClass.swift
+//  
+//
+//  Created by Dario Nikolic on 04/09/2020.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(CityCoreData)
+public class CityCoreData: NSManagedObject {
```

Comment:
I'm using this entity as read-only because its data ships with the app as pre-populated database so I didn't feel the need to implement firstOrCreate and populate methods